services.yaml

```diff
@@ -1791,7 +1791,7 @@ paths:
           $ref: "#/definitions/UnauthorizedError"
         404:
           description: NOT FOUND
-  /services/{service_id}/events/${event_id}/snapshot:
```

Comment:
didn't we say we'll leave this as last-snapshot?

Reply:
fixed